src/main/java/org/craftercms/studio/api/v2/annotation/RetryingRepositoryOperationAspect.java

```diff
@@ -63,13 +71,15 @@ public void setMaxSleep(int maxSleep) {
             "@annotation(org.craftercms.studio.api.v2.annotation.RetryingRepositoryOperation)")
     public Object doRetryingOperation(ProceedingJoinPoint pjp) throws Throwable {
         Method method = AopUtils.getActualMethod(pjp);
+        String repoPath = extractRepoPathFromPjp(pjp);
+
         Exception lastException;
 
         int numAttempts = 0;
         do {
             numAttempts++;
             try {
-				 // Execute the business code again
+                // Execute the business code again
                 if (numAttempts > 1) {
                     logger.debug("Retrying repository operation attempt " + (numAttempts - 1));
```

Comment:
Please don't use '+' in logs. You can use '{0}' and I'll fix it to '{}' when I switch it to slf4j. Or switch the file to slf4j with '{}'.

Reply:
Fixed, Thanks!